docs/ref/templates/builtins.txt

```diff
@@ -1832,7 +1832,7 @@ If ``value`` is the list ``['a', 'b', 'c']``, the output will be the string
 Safely outputs a Python object as JSON, wrapped in a ``<script>`` tag, ready
 for use with JavaScript.
 
-**Argument:** HTML "id" of the ``<script>`` tag.
+**Argument:** (optional) HTML "id" of the ``<script>`` tag.
```

Comment:
I guess this needs a `versionchanged`

Reply:
Good point. I've added a short sentence at the bottom of the whole `json_script` section.